docs/node-api.md

```
@@ -0,0 +1,176 @@
+
+### Including the headers
+
+Write the ecosystem-standard bare include, exactly as an addon for Node.js or the napi-ios runtime would:
+
+```c
+#include <node_api.h>
+```
+
+and add one header search path to your plugin target (podspec `pod_target_xcconfig` / Xcode build setting):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)